client/src/app/employee/employee-list/employee-list.component.html

```diff
@@ -22,20 +22,22 @@
       <td>{{ employee.employmentStartedOn | date }}</td>
       <td>
         <fa-icon
-          *ngIf="employee.status === 'ACTIVE'"
+          *ngIf="employee.status === 'Enabled'"
           [icon]="faSmileBeam"
         ></fa-icon>
         <fa-icon
-          *ngIf="employee.status === 'DISABLED'"
+          *ngIf="employee.status === 'Disabled'"
           [icon]="faSkull"
         ></fa-icon>
       </td>
       <td>
         <button
           [ngClass]="'btn btn-outline-warning btn-sm'"
-          [disabled]="employee.status === 'DISABLED'"
-          >Disable</button
+          (click)="updateEmployeeStatus(employee)"
```

Comment:
I typically name event handlers with an `on` prefix, so `onToggleEmployeeStatus(employee)`.

That when when looking at the methods in the component class you know which ones are being used for event handling in the template and you can organize those together.

Reply:
fixed